plugin-server/src/worker/ingestion/event-pipeline/4-processPersonsStep.ts

```diff
@@ -12,6 +12,13 @@ export async function processPersonsStep(
     personContainer: LazyPersonContainer
 ): Promise<StepResult> {
     const event = normalizeEvent(pluginEvent)
+
+    // TODO: Have the snapshot events pipeline be completely separate
+    // from all other events
+    if (event.event === '$snapshot') {
+        return runner.nextStep('prepareEventStep', event, personContainer)
```

Comment:
this is silly, we're doing this step just to do `normalizeEvent` but yup agreed fastest way

Reply:
Yeah I started this PR off by setting up a `runSnapshotEventPipeline`, but chances are I'd just leave more WIP for others